### PR TITLE
Add M1 Finance

### PR DESCRIPTION
**Pull request by SidB3**

### CODE REVIEW

File: img/m/m1finance.com.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1024 1024">
+   <circle cx="512" cy="512" r="512" style="fill:#12123d"/>
+   <path d="M390.1 695.14H276v-310l114.1 114.1ZM593 507.85l-199-199H276v46.19l234.89 234.91Zm-66-96.58 80.16 80.16L630 468.6v226h118v-384H627.77Z" style="fill:#fff"/>
+</svg>
```
Comment (Carlgo11) [inline img/m/m1finance.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1024 1024"><circle cx="512" cy="512" r="512" fill="#12123d"/><path fill="#fff" d="M390 695H276V385l114 114Zm203-187L394 309H276v46l235 235Zm-66-97 80 80 23-22v226h118V311H628Z"/></svg>
```

File: entries/m/m1finance.com.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      "domain": "m1finance.com",
+      "url": "https://www.m1finance.com",
+      "tfa": [
+        "totp"
+      ],
+      "documentation": "https://support.m1finance.com/hc/en-us/articles/221056367-Two-Factor-Authentication",
+      "keywords": [
+        "finance",
+        "investing"
+      ]
```
Comment (Carlgo11) [inline entries/m/m1finance.com.json:14]:
```suggestion
      ],
      "regions": [
        "us"
      ]
```

File: entries/m/m1finance.com.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+    "M1 Finance": {
+      "domain": "m1finance.com",
+      "url": "https://www.m1finance.com",
```
Comment (Carlgo11) [inline entries/m/m1finance.com.json]: The url tag isn't needed when the domain redirects to the same URL.
```suggestion
```

File: entries/m/m1finance.com.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+      "tfa": [
+        "totp"
+      ],
+      "documentation": "https://support.m1finance.com/hc/en-us/articles/221056367-Two-Factor-Authentication",
```
Comment (Carlgo11) [inline entries/m/m1finance.com.json]:
```suggestion
      "documentation": "https://support.m1finance.com/hc/en-us/articles/221056367",
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by SidB3** [comment]
Thanks! @Carlgo11 , Updated the pr with your suggestions.

**Review by Carlgo11 — APPROVED** [review]

**Comment by Carlgo11** [comment]
Thanks for the PR @Sid-Bezawada! :clap:

**Review by phallobst — APPROVED** [review]

**Pull request merged**